lib/lunchbot_web/live/order_live.html.heex

```diff
@@ -27,6 +27,12 @@
       <% end %>
     <% end %>
   <% else %>
-    <h1>No given menu currently.</h1>
+    <%= if assigns[:office_id] do %>
+      <h1>No given menu for this office currently.</h1>
+    <% else %>
+    <%= f = form_for :office_id, "#", phx_change: "office_select" %>
+        <%= label f, "Select an office" %>
+        <%= select f, :office, [nil: 0, "Providence": 1, "Boulder": 2], required: true %>
```

Comment:
If you don't want it hardcoded, Its also possible to define the selection options via a passed in arg in DB format like `Enum.map(@offices, &{&1.name, &1.id})`

[Phoenix.HTML.Form.select/4](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#select/4-examples)

Reply:
updated with a "Please select..." option, & using @offices instead of hard-coding.